mobile/android/logcat-helper.ts

```diff
@@ -10,7 +12,7 @@ export class LogcatHelper implements Mobile.ILogcatHelper {
 			private $staticConfig: Config.IStaticConfig) { }
 
 	public start(deviceIdentifier: string): void {
-		if(deviceIdentifier) {
+		if (deviceIdentifier && !this.isStarted) {
```

Comment:
this check should be per deviceIdentifier or we'll always have logs only for the first connected device (this will break Proton) :)